Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [evcc/evcc](https://hosted.weblate.org/projects/evcc/evcc/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/evcc/evcc/horizontal-auto.svg)

andig commented on i18n/zh-Hans.toml:
```diff
@@ -0,0 +1 @@
+
```
Was für eine Sprache ist das denn? Artefakt?

naltatis replied:
![Bildschirmfoto 2023-10-13 um 18 10 01](https://github.com/evcc-io/evcc/assets/152287/6236277e-704d-41eb-a297-5a82af40c592)

derkroesink committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Dutch)

Currently translated at 97.6% (291 of 298 strings)

Co-authored-by: Derk Roesink <d.roesink@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/nl/
Translation: evcc/evcc

amtssp committed:
Translated using Weblate (Danish)

Currently translated at 100.0% (298 of 298 strings)

Co-authored-by: amtssp <amtssp@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/da/
Translation: evcc/evcc

milotype committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (298 of 298 strings)

Co-authored-by: Milo Ivir <mail@milotype.de>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/hr/
Translation: evcc/evcc

mojocn committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 16.7% (50 of 298 strings)

Added translation using Weblate (Chinese (Simplified))

Co-authored-by: mojocn <neochau@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/zh_Hans/
Translation: evcc/evcc

RTTTC committed:
Translated using Weblate (Lithuanian)

Currently translated at 100.0% (298 of 298 strings)

Co-authored-by: RTTTC <94727758+RTTTC@users.noreply.github.com>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/lt/
Translation: evcc/evcc

kfussmann committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (298 of 298 strings)

Co-authored-by: kfussmann <karlfussmann@gmx.de>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/pt/
Translation: evcc/evcc

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (298 of 298 strings)

Co-authored-by: gallegonovato <fran-carro@hotmail.es>
Translate-URL: https://hosted.weblate.org/projects/evcc/evcc/es/
Translation: evcc/evcc

weblate committed:
Merge remote-tracking branch 'origin/master'

weblate committed:
Merge remote-tracking branch 'origin/master'

weblate committed:
Merge remote-tracking branch 'origin/master'

weblate committed:
Merge remote-tracking branch 'origin/master'

weblate committed:
Merge remote-tracking branch 'origin/master'

weblate committed:
Merge remote-tracking branch 'origin/master'

weblate committed:
Merge remote-tracking branch 'origin/master'

weblate committed:
Merge remote-tracking branch 'origin/master'

weblate committed:
Merge remote-tracking branch 'origin/master'

weblate committed:
Merge remote-tracking branch 'origin/master'

weblate committed:
Merge remote-tracking branch 'origin/master'

weblate committed:
Merge remote-tracking branch 'origin/master'

weblate committed:
Merge remote-tracking branch 'origin/master'

premultiply committed:
fix toml
